### PR TITLE
Fix various documentation issues

### PR DESCRIPTION
**Pull request by mifisignal**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Containment/forensics/README.md [hunk 1]
```diff
@@ -61,7 +65,7 @@ Since the script uploads content to an s3 bucket, the instance on which the scri
     "Version": "2012-10-17",
     "Statement": [
         {
-            "Sid": "VisualEditor0",
+            "Sid": "VisualEditor0",q
```
Comment (jicowan) [inline Containment/forensics/README.md]: did you add a "q" to the end of this line on purpose?

File: Containment/forensics/README.md [hunk 2]
```diff
@@ -78,19 +82,18 @@ kube-forensics obviates the need to specify the node on which the pod is running
...
+Once you've cordoned the node, you can begin rescheduling all the unaffected pods running on the node onto other nodes in the cluster. You can do this a couple different ways. The first way is to delete the unaffected pods. A cordoned node is marked as unschedulable. Pods that are deleted from condoned nodes will be rescheduled onto other nodes in the cluster. The second way is to apply a toleration to the compromised pod and then apply a taint to the node. This will evict all pods without a toleration for the taint from the node. For this exercise, we will be cordoning the node and deleting the unaffected pods.
 
 You can use the following bash script to delete the unaffected pods from the node. Be sure to replace the placeholder values `<POD_NAME>` and `<NODE_NAME>` before running the script. 
 
 ```bash
-#!/bin/bash
```
Comment (jicowan) [inline Containment/forensics/README.md:86]: What was the reason for deleting this?

**Review by jicowan — COMMENTED** [review]
@otterley Changes look good. Had a couple questions about a few of them.

File: Containment/forensics/README.md (hunk 1 above)
Comment (mifisignal) [reply Containment/forensics/README.md]: Nope! Fixing

File: Containment/forensics/README.md (hunk 2 above)
Comment (mifisignal) [reply Containment/forensics/README.md:86]: Also an error - putting back.

**Comment by mifisignal** [comment]
Updated per your feedback.

**Pull request merged**